### DISCUSSION TITLE
Gas meter reading

### DISCUSSION THREAD

**Discussion by NitramH** [jomjol/AI-on-the-edge-device#864] [Ideas]
Just an idea:

I have a gas meter near to the water meter. The gas meter has a magnet to trigger a reed contact.

It just needs an HW input with a trigger, which is reading the reed contact placed to the meter.
The idea is to avoid an additional ESP8266 and use the existing ESP32 watermeter.

I do not know the remaining unused performance and space of the watermeter.
The trigger could maybe confuse/interrupt the recognition algorithm?

What do you think, is this a possible feature?

Martin

**Comment by jomjol** [comment]
Technically possible, but the programming effort is much bigger than a 10 EUR ESP32. But if you implement it, I check, if it can be merged.

**Comment by NitramH** [comment]
I thought about your comment and you are right. 

Not only the HW is cheap, also the SW solution is already there.
Using Tasmota on a ESP8266 as switch for the reed contact is a well developed, fast and cheap solution.

Sometimes the easiest way is not directly visible :-)
